library/classes/thumbnail/Thumbnail.class.php

```diff
@@ -0,0 +1,192 @@
+<?php
+/**
+ * library/classes/thumbnail/Thumbnail.class.php Make thumbnail file using PHP GD functions.
+ *
+ * version 1 - just for images.
+ *
+ * LICENSE: This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 3
+ * of the License, or (at your option) any later version.
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://opensource.org/licenses/gpl-license.php>;.
+ *
+ * @package OpenEMR
+ * @author  Amiel Elboim <amielel@matrix.co.il>
+ * @link    http://www.open-emr.org
+ */
+
+class Thumbnail
+{
+
+    //max size of thumbnail
+    public $max_size = 100;
+    public $thumbnail_type = 'jpg';
```

Comment:
Yes. Thumbnail type does not depend on content type. In future we can add configuration gif/png/jpg if we choose to.